apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/java/file/InputObjectBuilder.kt

```diff
@@ -52,17 +53,17 @@ internal class InputObjectBuilder(
       return this
     } else {
       val builderFields = inputObject.fields.map {
-        context.layout.propertyName(it.name) to context.resolver.resolveIrType(it.type)
+        FieldSpec.builder(context.resolver.resolveIrType(it.type).withoutAnnotations(), context.layout.propertyName(it.name))
```

Comment:
Refactored `Builder` to take a list of `FieldSpec` instead of maps

Reply:
Much better 👍 

Nit: should this class be named `BuilderBuilder`? 🙃

Reply:
> should this class be named BuilderBuilder?

Actually, yes I think so! Will rename :)